.github/workflows/test.yaml

```diff
@@ -61,23 +67,24 @@ jobs:
         run: >-
           touch .env.development;
           touch .env;
-          docker-compose pull web
+          docker pull ${{ env.REGISTRY }}/${{ env.REPO_LOWER }}:${{ env.TAG }}
```

Comment:
docker-compose commands are not used in our self-hosted runner. I don't really know why, but this solved the issue.

Reply:
this was not the way